lib/rubocop/changes/checker.rb

```diff
@@ -59,7 +60,24 @@ def ruby_changed_files
       end
 
       def rubocop
-        Shell.run("rubocop --force-exclusion -f j #{ruby_changed_files.join(' ')}")
+        shell_command = ['rubocop']
```

Comment:
I think it could be simpler like this:

```ruby
      shell_command = [
        'rubocop',
        exclussion_modifier,
        formatter_modifier,
        auto_correct_modifier
      ].compact.join(' ')

      def exclussion_modifier
        '--force-exclusion'
      end

      def formatter_modifier
        "-f j #{ruby_changed_files.join(' ')}"
      end

      def auto_correct_modifier
        '-a' if @auto_correct
      end
```